experiments/slurm_base_char_sc.sh

```
@@ -0,0 +1,44 @@
+#SBATCH --mem=48gb
+#SBATCH --gpus=1
+#SBATCH --output=%x_%a_%j.log # %x is job-name, %j is job id, %a is array id
+#SBATCH --array=0-3
+
+# Submit with e.g. --export=TASK_NAME=mortality24,MODEL_NAME=LGBMClassifier
+# Basic experiment variables, please exchange [INSERT] for your experiment parameters
+
+TASK=BinaryClassification # BinaryClassification
+YAIB_PATH=/home/vandewrp/projects/YAIB #/dhc/home/robin.vandewater/projects/yaib
+EXPERIMENT_PATH=../yaib_logs/${TASK_NAME}_experiment
+DATASET_ROOT_PATH= /sc-projects/sc-proj-cc08-cassandra/Prospective_Preprocessed/yaib_format #data/YAIB_Datasets/data
```

Shellcheck (0.10.0)

[warning] Remove space after = if trying to assign a value (for empty string, use var='' ... ).

(SC1007)